src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfilesController.php

```diff
@@ -0,0 +1,161 @@
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Controller\Admin\Configure\AdvancedParameters;
+
+use PrestaShop\PrestaShop\Core\Search\Filters\ProfilesFilters;
+use PrestaShopBundle\Controller\Admin\FrameworkBundleAdminController;
+use PrestaShopBundle\Security\Annotation\AdminSecurity;
+use PrestaShopBundle\Security\Annotation\DemoRestricted;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Class ProfilesController is responsible for displaying the
+ * "Configure > Advanced parameters > Team > Profiles" page.
+ */
+class ProfilesController extends FrameworkBundleAdminController
+{
+    /**
+     * Show profiles listing page.
+     *
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller'))")
+     *
+     * @Template("@PrestaShop/Admin/Configure/AdvancedParameters/Profiles/profiles.html.twig")
+     *
+     * @param ProfilesFilters $filters
+     *
+     * @return array
+     */
+    public function indexAction(ProfilesFilters $filters)
+    {
+        $profilesGridFactory = $this->get('prestashop.core.grid.factory.profiles');
+        $gridPresenter = $this->get('prestashop.core.grid.presenter.grid_presenter');
+
+        return [
+            'layoutHeaderToolbarBtn' => [
+                'add' => [
+                    'href' => $this->getAdminLink('AdminProfiles', ['addprofile' => '']),
+                    'desc' => $this->trans('Add new profile', 'Admin.Advparameters.Feature'),
+                    'icon' => 'add_circle_outline',
+                ],
+            ],
+            'enableSidebar' => true,
+            'help_link' => $this->generateSidebarLink('AdminProfiles'),
+            'grid' => $gridPresenter->present($profilesGridFactory->getGrid($filters)),
+        ];
+    }
+
+    /**
+     * Used for applying filtering actions.
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function searchAction(Request $request)
+    {
+        $definitionFactory = $this->get('prestashop.core.grid.definition.factory.profiles');
+        $definitionFactory = $definitionFactory->getDefinition();
+
+        $gridFilterFormFactory = $this->get('prestashop.core.grid.filter.form_factory');
+        $searchParametersForm = $gridFilterFormFactory->create($definitionFactory);
```

Comment:
should be createForm, I've forgot to update it :/

Reply:
@PierreRambaud where were you when we had discussion with @mickaelandrieu about that for other services. -_- i needed your backup. :p

Reply:
Didn't see the discussion... Too many things to do :'(

Reply:
in short, it was about `GridDefinitionFactoryInterface::create()` => `GridDefinitionFactoryInterface::getDefinition()`

Reply:
Always call me when you want someone to do mess with naming :smile:

Reply:
we may still fix that untill 1.7.5 release if you get @mickaelandrieu on your side. :smile:

Reply:
@matks wdyt too?

Reply:
I'm sorry, I'm going to disappoint all of you ^^ : I think both names are right. One is more redundant, but both names provide the required information ("what is this function doing ?") very accurately so I do not care.

Reply:
I think it's too late to change the name, but if we can I stand for `getForm()` for consistency with all others functions in the Grid component.